infra/scripts/coverage.sh

```diff
@@ -44,6 +44,7 @@ generate_pkg_cover_data() {
 
     for pkg in "$@"; do
         f="$workdir/$(echo $pkg | tr / -).cover"
+        go test -v -i -covermode="$mode" -coverprofile="$f" "$pkg"
```

Comment:
2 go test invocations for the same pkg.  Does that mean the tests will run twice now?